### DISCUSSION TITLE
gpx to strava

### DISCUSSION THREAD

**Discussion by sauchye** [yihong0618/running_page#322] [Q&A] [answered]
刚把tcx格式同步到strava，同步完后发现strava总里程没有变化，这里有延时么？
还有gpx不知如何同步strava，感谢老哥！
```
python3 scripts/tcx_to_strava_sync.py xx xx xx

Need to load all tcx files maybe take some time
False
Access ok
Start syncing
..................................................%
```

**Comment by yihong0618** [comment, marked as answer]
python3 scripts/tcx_to_strava_sync.py xx xx xx --all
1. 懂了，starva 是通过时间戳找下一个决定的， --all 参数试试，如果不行我找时间更新下
2. gpx 暂时不支持，我加一下。

**Reply by yihong0618** [reply]
@sauchye

**Reply by sauchye** [reply]
是否可以将数据同步到非国区的Garmin账户呢，然后Garmin同步到Strava

**Reply by yihong0618** [reply]
可以的。

**Comment by yihong0618** [comment]
@sauchye 支持了 gpx to strava 可以尝试下了。因为这个是按时间戳找下一个的，你可能需要加上 `--all` 参数
